### PR TITLE
refactor : optimize Appwrite Client instantiation

### LINKED ISSUE #128: Centralize Appwrite Client Instance
Currently we are initializing appwrite again and again in different controllers. A single instance can be created and used wherever required.

### PR DESCRIPTION
**Pull request by ShamanthOP**
Closes #128 

Refactored Appwrite client into it's own service where it is instantiated only once.

### CODE REVIEW

File: lib/services/appwrite_service.dart
Comment (chandansgowda) [inline lib/services/appwrite_service.dart:1]: Can we get databases and realtime instances also from here ? What would you suggest ?

Comment (ShamanthOP) [reply lib/services/appwrite_service.dart:1]: Yeah we can do that as well. Shall I give it a go?

Comment (chandansgowda) [reply lib/services/appwrite_service.dart:1]: Yes

**Comment by chandansgowda** [comment]
@ShamanthOP Please resolve conflicts

**Comment by ShamanthOP** [comment]
> @ShamanthOP Please resolve conflicts

Resolved.

**Comment by chandansgowda** [comment]
Great job @ShamanthOP 🚀

**Pull request merged**